blockchain/sync_manager/src/sync.rs

```diff
@@ -15,14 +15,16 @@ use libp2p::core::PeerId;
 use message::{Message, MsgMeta};
 use num_bigint::BigUint;
 use raw_block::RawBlock;
-use state::{HamtStateTree, StateTree};
+use state_manager::StateManager;
+use state_tree::{HamtStateTree, StateTree};
```

Comment:
side note the HamtStateTree usage here can be removed by referencing the state tree as the trait and not the specific implementation